src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -505,4 +505,44 @@ public function update_order(Request $request)
             $i->save();
         }
     }
+
+    public function relation(Request $request)
+    {
+        $slug = $this->getSlug($request);
+        $page = $request->input('page');
+        $on_page = 50;
+        $search = $request->input('search', false);
+
+        $dataType = Voyager::model('DataType')->where('slug', '=', $slug)->first();
+        $relationships = $this->getRelationships($dataType);
+
+        //$this->authorize('edit', $dataTypeContent);
+
+        foreach ($dataType->editRows as $key => $row) {
+            if ($row->field === $request->input('type')) {
+                $details = json_decode($row->details);
+                $options = $details;
```

Comment:
`$details doesn't seem to be used anywhere else.  Can we combine these 2 lines into one?